Makefile

```diff
@@ -182,6 +182,10 @@ staging-dotenv-file: guard-VAULT_ADDR guard-VAULT_TOKEN vault-installed
 	@ echo ENVIRONMENT=staging >> .env
 	@ echo SUBDOMAIN=staging. >> .env
 	@ echo RML_MAPPER_PATH=${RML_MAPPER_PATH} >> .env
+	@ echo ELK_HOST=localhost >> .env
```

Comment:
This shall be handled by Dragos in Vault

Reply:
He can updated this Make targets when ELK is ready